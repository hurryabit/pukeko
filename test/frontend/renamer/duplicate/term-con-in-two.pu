data Typ1 =
  | Con
data Typ2 =
  | Con
