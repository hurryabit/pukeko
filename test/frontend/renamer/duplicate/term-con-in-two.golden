test/frontend/renamer/duplicate/term-con-in-two.pu:4:5: duplicate data constructor: Con
